src/views/index_view.js

```diff
@@ -28,6 +33,10 @@ export default BaseView.extend({
     this.renderSubview(new IconView());
```

Comment:
That's an option, but some of the images that affect document height are loaded asynchronously.